docs/tutorial/common_hpc_dependencies.rst

```diff
@@ -101,9 +101,9 @@ or ``CUDA_HOST_COMPILER`` in previous versions.  If you do not call
 ``enable_language(CUDA)``, BLT will set the appropriate host compiler variable
 for you and enable the CUDA language.
 
-Here is a snippet with these settings for LLNL's Pascal Cluster:
+Here is a snippet with these settings for LLNL's Lassen Cluster:
```

Comment:
This page is missing a similar fix above and this isnt work:

https://llnl-blt.readthedocs.io/en/feature-chapman39-update-host-configs/tutorial/common_hpc_dependencies.html